VideoLocker/res/layout/fragment_course_unit_only_on_youtube.xml

```diff
@@ -0,0 +1,45 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <org.edx.mobile.view.custom.IconImageViewXml
+        android:layout_width="90dp"
+        android:layout_height="90dp"
```

Comment:
These should go into the `dimens.xml` file, or we could just use `large_indicator_icon_width` & `large_indicator_icon_height` which are `100dp` each.